### PR TITLE
updated the code for team_slave in ifconfig file

### LINKED ISSUE #1626: ifcfg parser to support for MASTER and TEAM_MASTER keys slave type
We need to update the ifcfg parser to support TEAMING and BONDING slave type in the configuration file, so, that we can use MASTER' and 'TEAM_MASTER' keys in raw format.

For ex-  `obj['MASTER']="'bond0'"` or `obj['TEAM_MASTER']="'team0'"`

### PR DESCRIPTION
**Pull request by ghost**
Fix #1626

### CODE REVIEW

File: insights/parsers/tests/test_ifcfg.py
Comment (xiangce) [inline insights/parsers/tests/test_ifcfg.py]: The value of this `IFCFG_PATH_NAMED_TEAM_MASTER` is the same to `IFCFG_PATH_NAMED_BOND_MASTER`. Better just reserve one and rename it to e.g. `IFCFG_PATH_1`

**Review by xiangce — CHANGES_REQUESTED** [review]
Hi @rasrivas-redhat 

What's the reason that you keep the RAW value of the "TEAM_MASTER" ?
Please open an issue for it [here](https://github.com/RedHatInsights/insights-core/issues) at first.

BTW, please explain the reason for keeping the RAW value for "MASTER" as well in the same issue. Thanks.

**Comment by rasrivas-redhat** [comment]
@xiangce 
First, we updated the code for "Bonding slave type" which we mentioned as MASTER="xxx" in config file but for "teaming slave type" we need to mention as TEAM_MASTER="xxx" in config file.

File: insights/parsers/tests/test_ifcfg.py
Comment (xiangce) [inline insights/parsers/tests/test_ifcfg.py]: You need to remove the `IFCFG_PATH_NAMED_BOND_MASTER` and `IFCFG_PATH_NAMED_BOND_MASTER`

Comment (rasrivas-redhat) [reply insights/parsers/tests/test_ifcfg.py]: @xiangce removed the `IFCFG_PATH_NAMED_TEAM_MASTER` and `IFCFG_PATH_NAMED_BOND_MASTER` and updated the test case for `IFCFG_PATH_ETH1` and `IFCFG_PATH_ETH2` path which are used in test case

**Review by bfahr — APPROVED** [review]
Approved pending @xiangce approval.

**Review by xiangce — APPROVED** [review]

**Pull request merged**
